sw/device/lib/testing/test_framework/ottf_main.c

```diff
@@ -82,6 +87,13 @@ void _ottf_main(void) {
     LOG_INFO("Running %s", kOttfTestConfig.file);
   }
 
+  // Initialize a global random number generator testutil context.
+  dif_rv_core_ibex_t rv_core_ibex;
+  CHECK_DIF_OK(dif_rv_core_ibex_init(
+      mmio_region_from_addr(TOP_EARLGREY_RV_CORE_IBEX_CFG_BASE_ADDR),
+      &rv_core_ibex));
+  rand_testutils_rng_ctx = rand_testutils_init(&rv_core_ibex);
```

Comment:
Do we think it's worth adding a series of CHECKs here to ensure the entropy complex is initialized correctly? It would be 4 CSR reads and checks to ensure which ever ROM is in place (or if a test is run at a later boot stage) configures entropy complex like it is [currently configured](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/testing/test_rom/test_rom_start.S;drc=b602e150472bd499651e2e6b1ddbfcbb4d852232;l=177) in the test ROM.

Four reads/checks might be too much overhead, just thinking out loud here :) I'm ok with either way.

Reply:
I don't think it's needed, given both ROMs do initialize the entropy SS in the start up code itself (I am sure something else will fail if they don't). If you disagree, then I will prefer addressing it in a different PR, if that is ok with you.

Reply:
Sg, I OK with it as is. Was just throwing this out there to partially address @tjaychen 's [comment above](https://github.com/lowRISC/opentitan/pull/14416#discussion_r952752423)